### ISSUE TITLE
[Chains Ovewrite] how Do I get a list of all chains?

### ISSUE THREAD

**Issue by dann254** [taylorjdawson/eth-chains#11]
Looking at this line, Looks like the chains export is overwritten 🤔 

[https://github.com/taylorjdawson/eth-chains/blob/e898ed72e182ca1384aba09be94e268098837d61/index.ts#L2](https://github.com/taylorjdawson/eth-chains/blob/e898ed72e182ca1384aba09be94e268098837d61/index.ts#L2)

**Comment by dann254** [comment]
Just noticed `ChainId` and `ChainName` contain the lists when no parameter is passed in. 

But it would be nice to have the entire object available.

**Comment by taylorjdawson** [comment]
@dann254 Sorry for the delay on this! What about doing this instead?
```ts
const chains from 'eth-chains'
console.log(chains.all())
// { ... }
```

**Comment by taylorjdawson** [comment]
addressed in #14
